include/swift/AST/NameLookup.h

```diff
@@ -278,7 +278,11 @@ enum class DeclVisibilityKind {
   ///   }
   /// }
   /// \endcode
-  MemberOfProtocolImplementedByCurrentNominal,
+  MemberOfProtocolNotImplementedByCurrNominal,
+
+  /// Declaration is a derived requirement of a protocol conformed to by the
+  /// immediately enclosing nominal decl. Member of a synthesized conformance.
+  MemberOfProtocolDerivedByCurrNominal,
```

Comment:
Nitpick `Curr` is not consistent with `MemberOfCurrentNominal`. Could you use `Current`?

Reply:
It was getting a bit too long, but sure!

Reply:
Agreed, `Current` please.